index.html

```diff
@@ -1,284 +1,309 @@
+      <div>
+        <h1 class="intro">I'm Walid <br />Glad to see You!</h1>
+      </div>
+      <div>
+        <p class="about-me">
+          I’m a software developer! I can help you 
+          build a product , feature or website Look 
+          through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don’t 
+          hestiate to contact me.
+        </p>
+      </div>
+      <div class="contact-me">
+        <h2 class="connect">let's connect</h2>
+      </div>
+      <div>
+        <ul class="buttons">
+          <li><a href=""><img src="./assets/Github.svg" class="contact-btn" alt="Github" /></li></a>
+          <li><a href=""><img src="./assets/Twitter.svg" class="contact-btn" alt="Twitter" /></li></a>
+          <li><a href=""><img src="./assets/medium.svg" class="contact-btn" alt="Medium" /></li></a>
+          <li><a href=""><img src="./assets/Vector.svg" class="contact-btn" alt="" /></li></a>
         </ul>
-        </div>
-      </section>
+      </div>
+    </section>
 
-      <section class="card">
+    <section class="card">
       <div class="grid-container">
         <div class="first-card">
           <img class="card-img" src="/assets/First Snapshoot Portfolio.svg" alt="Tonic">
-          <div class="my-project"> 
-
-          <h2 class="project-name">Tonic</h2>
-          <div class="experience">
-          <span class="tag1">CANOPY</span>
-          <span class="bullet"></span>
-          <span class="tag">Back End Dev</span>
-          <span class="bullet"></span>
-          <span class="tag"> 2015</span>
-          
-        </div>
-       
-        <div class="descreption" >
-          <p> A daily selection of privately
-              personalized reads; no accounts or 
-              sign-up required.</p>
-          </div>
+          <div class="my-project">
+
+            <h2 class="project-name">Tonic</h2>
+            <div class="experience">
+              <span class="tag1">CANOPY</span>
+              <span class="bullet"></span>
+              <span class="tag">Back End Dev</span>
+              <span class="bullet"></span>
+              <span class="tag"> 2015</span>
+
+            </div>
+
+            <div class="descreption">
+              <p> A daily selection of privately
+                personalized reads; no accounts or
+                sign-up required.</p>
+            </div>
 
             <div class="languages">
               <ul>
-                <li  class="language-tag">html</li>
-                <li  class="language-tag">css</li>
-                <li  class="language-tag">Javascript</li>
+                <li class="language-tag">html</li>
+                <li class="language-tag">css</li>
+                <li class="language-tag">Javascript</li>
               </ul>
             </div>
-             <button class="see-project">See Project</button>
+            <button class="see-project">See Project</button>
```

Comment:
- [ ] As mentioned above one of the goal for this project is to parse a figma template as close as possible, how about implementing the button interactions to match the template design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213237749-f5f24f52-a826-4efd-b2ca-a8193d61152c.png)

- [ ] Please also fix the border-raius property to match the original template :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213241372-d5f96aa0-a1f2-44ca-b847-e059c893c7ed.png)